Automattic/wp-calypso
# Post Editor: Wire up the dialog opt-in button to set the blog's editor to Gutenberg. [WIP]

Upcoming API work will add an endpoint for setting a blog to use the Gutenberg editor. This PR stubs out an action to consider while the API work happens.

**Testing instructions**

- Load `/post/:site` and then "Learn more" in the sidebar's Gutenberg visual.
- In the dialog, click the "Try the new editor" button.
- Verify a successful call to `TBD`.
- Verify the test site has the correct blog sticker.
